detection_rules/devtools.py

```diff
@@ -105,26 +103,29 @@ def build_release(config_file, update_version_lock: bool, generate_navigator: bo
     if verbose:
         click.echo(f'[+] Building package {config.get("name")}')
 
-    package = Package.from_config(config, verbose=verbose, historical=add_historical)
+    package = Package.from_config(config, verbose=verbose)
 
     if update_version_lock:
         default_version_lock.manage_versions(package.rules, save_changes=True, verbose=verbose)
     package.save(verbose=verbose)
 
-    if add_historical:
-        previous_pkg_version = find_latest_integration_version("security_detection_engine", "ga",
-                                                               registry_data['conditions']['kibana.version'].strip("^"))
-        sde = SecurityDetectionEngine()
-        historical_rules = sde.load_integration_assets(previous_pkg_version)
-        historical_rules = sde.transform_legacy_assets(historical_rules)
-
+    previous_pkg_version = find_latest_integration_version("security_detection_engine", "ga",
+                                                           registry_data['conditions']['kibana.version'].strip("^"))
+    sde = SecurityDetectionEngine()
+    historical_rules = sde.load_integration_assets(previous_pkg_version)
+    historical_rules = sde.transform_legacy_assets(historical_rules)
+    package.add_historical_rules(historical_rules, registry_data['version'])
+    click.echo(f'[+] Adding historical rules from {previous_pkg_version} package')
+
+    # TODO: adjust code according to new security docs initiative
+    # NOTE: this is a temporary fix to avoid breaking the build
+    # NOTE: this code was introduced but is not used with the current implementation
```

Comment:
I would just reframe this as a stopgap solution until we tackle the doc migration.

Reply:
done.